env.example

```diff
@@ -59,6 +59,9 @@ ELASTIC_MAX_RESULTS=50
 # Delay to check connectivity with Elasticsearch in ms
 ELASTIC_DELAY_MS=10000
 
+# Kibana info
+KIBANA_PORT=5601
```

Comment:
What if I want to run this on a different box/VM than my localhost locally?

Reply:
Got it, I'll add `KIBANA_URL=` to the env.example